gravityflow/gravityflow
# Fix merge tags backwards compatibility

The PR fixes an issue introduced in 2.1 with the assignee refactor. Gravity_Flow_Assignee->replace_variables() is called in favour of Gravity_Flow_Step->replace_variables() so merge tags that are not registered with Gravity_Flow_Merge_Tags will not be replace. For example, the merge tags in the Form Submission step.

This fix is just a temporary fix for backward compatibility. A bit of refactoring is required before Gravity_Flow_Step->replace_variables() can be deprecated.

**Testing instructions**
- Set up a form submission step assigned to an email field. Make sure the assignee email has the {workflow_form_submission_link} merge tag.
- Submit the form using the master branch and the merge tag will not be replaced.
- Appy this PR and submit the form again and the merge tag will be replaced.
